src/fannkuch_redux.rs

```diff
@@ -19,7 +43,7 @@ const NUM_BLOCKS: u32 = 24;
 fn fannkuch(n: i32) -> (i32, i32) {
```

Comment:
This function is using `cfg`'d code without itself being `cfg`'d, so it will fail to compile whenever `U8x16` isn't defined.

Reply:
I know. At the moment I have two options:

1. leave it as is and have it fail on anything but `x86_64` with `sse2`.
2. create alternative SIMD-less implementations that will run much slower, and take up src size.

Perhaps I'll leave 1 to the game and submit 2 here later?

Reply:
Require a specific architecture is fine. Better to not complexity the program for portability. But doing it properly is a must.

Maybe the whole program should be cfg-ified?